packages/babel-parser/src/parser/location.js

```diff
@@ -98,6 +98,11 @@ export const Errors = Object.freeze({
   MissingUnicodeEscape: "Expecting Unicode escape sequence \\uXXXX",
   MixingCoalesceWithLogical:
     "Nullish coalescing operator(??) requires parens when mixing with logical operators",
+  ModuleAttributesIncorrectValue:
```

Comment:
suggestion: the `ModuleAttributeInvalidValue` seems more consistent to me with other naming conventions.

Reply:
Updated the code with `ModuleAttributeInvalidValue`